IBAnimatableTests/FillDesignableTests.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  FillDesignableTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 4/25/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
+
+class FillDesignableTests: XCTestCase {
+
+  override func setUp() {
+    super.setUp()
+  }
+
+  override func tearDown() {
+    super.tearDown()
+  }
+
+  // MARK: - testFillColor
+
+  func testFillColor() {
+    helper_testFillColor_UIView(element: AnimatableView())
+    helper_testFillColor_UIView(element: AnimatableScrollView())
+    helper_testFillColor_UIView(element: AnimatableButton())
+    helper_testFillColor_UIView(element: AnimatableCheckBox())
+    helper_testFillColor_UIView(element: AnimatableImageView())
+    //helper_testFillColor_UIView(element: AnimatableStackView()) // broken see #461
+    helper_testFillColor_UIView(element: AnimatableTableView())
+    helper_testFillColor_UIView(element: AnimatableTextField())
+    helper_testFillColor_UIView(element: AnimatableTextView())
+    helper_testFillColor_Cell(element: AnimatableTableViewCell())
+    helper_testFillColor_Cell(element: AnimatableCollectionViewCell())
+  }
+
+  private func helper_testFillColor_UIView(element: MockUIViewFillDesignable) {
+    element.fillColor = .red
+    XCTAssertEqual(element.backgroundColor, .red)
+  }
+
+  private func helper_testFillColor_Cell(element: MockCellFillDesignable) {
+    element.fillColor = .red
+    XCTAssertEqual(element.backgroundColor, .red)
+    XCTAssertEqual(element.contentView.backgroundColor, .red)
+  }
+
+  // MARK: - testOpacity
+
+  func testOpacity() {
+    helper_testOpacity(element: AnimatableView())
+    helper_testOpacity(element: AnimatableScrollView())
+    helper_testOpacity(element: AnimatableButton())
+    helper_testOpacity(element: AnimatableCheckBox())
+    helper_testOpacity(element: AnimatableImageView())
+    //helper_testOpacity(element: AnimatableStackView()) // broken see #461
```

Comment:
#461 is regarding `AnimatableStackView` being a non-drawing view
So changing things like the `backgroundColor`, `opacity`, won't take effect.
My goal is to re-evaluate this class when I have higher test coverage for it.
This way we can see what else is not taking affect on it.